components/state/state.go

```diff
@@ -97,7 +102,12 @@ INSERT OR REPLACE INTO %s (%s, %s) VALUES (?, ?);
 		ColumnMachineID,
 		ColumnUnixSeconds,
 	)
-	if _, err := dbRW.ExecContext(ctx, query, uid, time.Now().UTC().Unix()); err != nil {
+
+	start = time.Now()
+	_, err = dbRW.ExecContext(ctx, query, uid, time.Now().UTC().Unix())
```

Comment:
do we need `UTC()` here? does `time.Now().Unix()` work?

Reply:
In the past, we had some providers using Singapore timezone lol